### PR TITLE
Increase client/server update rate to 125Hz

### PR DESCRIPTION
**Pull request by voidanix**
Currently the update rates are at 25Hz, but it's 2020

Increase them to 125Hz

### CODE REVIEW

**Comment by shacknetisp** [comment]
This should be converted to a constant instead of a magic number.

File: src/game/game.h [hunk 1]
```diff
@@ -59,6 +59,10 @@ struct enttypes
     bool    noisy,  syncs,  resyncs,    syncpos,    synckin;
     const char *name,           *attrs[13];
 };
+
+// delay in milliseconds between updates for client/server
```
Comment (TheAssassin) [inline src/game/game.h:63]: The description is not entirely accurate. It's the minimum interval for updates. All we guarantee is that updates are not sent more often than in this interval (i.e., we guarantee a lower boundary for this delay), but we cannot ensure updates are sent at this interval (e.g., with low frame rates, updates are sent much less often).
We just increases the *possible* update rate in this PR.

File: src/game/bomber.cpp [hunk 2]
```diff
@@ -626,10 +626,10 @@ namespace bomber
                 vec pos = f.pos();
                 f.distance += f.droploc.dist(pos);
                 f.droploc = pos;
-                if(f.lastowner && (f.lastowner == &game::player1 || f.lastowner->ai) && f.proj && (!f.movetime || totalmillis-f.movetime >= 40))
+                if(f.lastowner && (f.lastowner == &game::player1 || f.lastowner->ai) && f.proj && (!f.movetime || totalmillis-f.movetime >= UPDATERATE))
```
Comment (MoonPadUSer) [inline src/game/bomber.cpp:629]: please just add a space between `if` and `(`

File: src/game/capture.cpp [hunk 3]
```diff
@@ -582,10 +582,10 @@ namespace capture
             if(f.droptime)
             {
                 f.droploc = f.pos();
-                if(f.lastowner && (f.lastowner == &game::player1 || f.lastowner->ai) && f.proj && (!f.movetime || totalmillis-f.movetime >= 40))
+                if(f.lastowner && (f.lastowner == &game::player1 || f.lastowner->ai) && f.proj && (!f.movetime || totalmillis-f.movetime >= UPDATERATE))
```
Comment (MoonPadUSer) [inline src/game/capture.cpp:585]: please just add a space between `if` and `(`

File: src/game/client.cpp [hunk 4]
```diff
@@ -1883,7 +1883,7 @@ namespace client
     void c2sinfo(bool force) // send update to the server
     {
         static int lastupdate = -1000;
-        if(totalmillis-lastupdate < 40 && !force) return;    // don't update faster than 25fps
+        if(totalmillis - lastupdate < UPDATERATE && !force) return;    // don't update faster than 125fps
```
Comment (MoonPadUSer) [inline src/game/client.cpp:1886]: please just add a space between `if` and `(`

**Pull request merged**